examples/cosmos-etcd/readme.md

```diff
@@ -1,9 +1,58 @@
 # Kubernetes on Cosmos
+This example deployment allows you to use aks-engine to bootstrap a Kubernetes cluster on Azure that uses [Azure Cosmos](https://azure.microsoft.com/services/cosmos-db/) instead of locally installed and configured etcd.
```

Comment:
all references to the project should be `AKS Engine`, all references to the command `aks-engine` for consistency.

Reply:
```suggestion
This example deployment allows you to use AKS Engine to bootstrap a Kubernetes cluster on Azure that uses [Azure Cosmos](https://azure.microsoft.com/services/cosmos-db/) instead of locally installed and configured etcd.
```